shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/UpstreamCheckService.java

```diff
@@ -147,22 +147,22 @@ public UpstreamCheckService(final SelectorMapper selectorMapper,
         this.scheduledTime = Integer.parseInt(props.getProperty(Constants.SCHEDULED_TIME, Constants.SCHEDULED_TIME_VALUE));
         this.registerType = shenyuRegisterCenterConfig.getRegisterType();
         zombieRemovalTimes = Integer.parseInt(props.getProperty(Constants.ZOMBIE_REMOVAL_TIMES, Constants.ZOMBIE_REMOVAL_TIMES_VALUE));
-        if (REGISTER_TYPE_HTTP.equalsIgnoreCase(registerType)) {
-            setup();
-        }
     }
 
     /**
      * Set up.
      */
     public void setup() {
-        if (checked) {
-            this.fetchUpstreamData();
-            executor = new ScheduledThreadPoolExecutor(1, ShenyuThreadFactory.create("scheduled-upstream-task", false));
-            scheduledFuture = executor.scheduleWithFixedDelay(this::scheduled, 10, scheduledTime, TimeUnit.SECONDS);
-
-            ThreadFactory requestFactory = ShenyuThreadFactory.create("upstream-health-check-request", true);
-            invokeExecutor = new ScheduledThreadPoolExecutor(this.scheduledThreads, requestFactory);
+        if (REGISTER_TYPE_HTTP.equalsIgnoreCase(registerType)) {
```

Comment:
change the conditon `if (REGISTER_TYPE_HTTP.equalsIgnoreCase(registerType) && checked)) {`